lib/pubsub/subscription.js

```diff
@@ -296,32 +300,53 @@ Subscription.prototype.delete = function(callback) {
  *   // message.id = ID used to acknowledge its receival.
  *   // message.data = Contents of the message.
  * });
+ *
+ * //-
+ * // Limit the results.
+ * //-
+ * subscription.pull({ maxCount: 3 }, function(err, messages) {});
  */
 Subscription.prototype.pull = function(options, callback) {
   var that = this;
-  // TODO(jbd): Should not be racing with other pull.
+
   if (!callback) {
     callback = options;
     options = {};
   }
+
+  var batch = util.is(options.maxCount, 'number');
```

Comment:
Thinking `.pull` should default to `pullBatch`, we just need to decide what to set `maxCount` to (`maxEvents` as the API calls it).

The only way a user would be able to pull a single message would be with `sub.pull({ maxCount: 1 })`